packages/disploy/README.md

```diff
@@ -17,56 +17,177 @@
 
 </div>
 
-> **Warning**: We're still in development, and the packages published are not up to date. They're just early builds that we've published to reserve our package names on NPM.
+> **Warning**: We're still in development, and packages are published to NPM every 12 hours to the `@dev` tag.
 
-Disploy is a framework for building Discord bots with ease. It's designed to make it easy to build, test and deploy Discord bots.
+Disploy is a library for building HTTP interaction-based Discord bots with ease. It's designed to make it easy to build, test and deploy Discord bots.
 
-## It's as easy as 1, 2, 3, 4
+# Features
 
-### #1
+Disploy features a [library](#library) and an opinionated [framework](#framework) with tooling inspired by Next.js.
+
+## Library
+
+> Disploy does not come included with a "server", that's up to you to implement. We have a [guide](https://disploy.dev/docs/Reference/framework-less/) showcasing you how to do so with Express (Node.js) and Deno's inbuilt server.
+
+In this readme, we have supplied you with a more slimmed-down guide on usage with Next.js as your server.
+
+### Usage with Next.js
+
+The API entry point
+
+```ts
+// Entrypoint - pages/api/interactions.ts
+import { createNextAdapter } from 'disploy';
+import { App } from '../../lib/main';
+
+export default createNextAdapter(SafecordApp);
+```
+
+> **Note**: An "adapter" is a function that transforms requests from your server implementation of choice and creates a `TRequest` that's fed into `App#router#entry` which returns a `Promise<TResponse>` which your adapter should transform and return to Discord.
+
+Setting up the Disploy App
+
+```ts
+// Main Bot - lib/core/main.ts
+import { App } from 'disploy';
+import commands from './commands/commands';
+
+const clientId = process.env.DISCORD_CLIENT_ID;
+const token = process.env.DISCORD_TOKEN;
+const publicKey = process.env.DISCORD_PUBLIC_KEY;
+
+if (!clientId || !token || !publicKey) {
+	throw new Error('Missing environment variables');
+}
+
+export const SafecordApp = new App({
+	logger: {
+		debug: true,
+	},
+});
+
+SafecordApp.start({
+	clientId,
+	token,
+	publicKey,
+});
+
+for (const command of commands) {
+	SafecordApp.commands.registerCommand(command);
+}
+```
+
+Setting up an array of commands
+
+```ts
+// Command Array - lib/core/commands/commands.ts
+import Ping from './core/ping';
+
+const c = [Ping];
+
+export default c;
+```
+
+Example command
+
+```ts
+import type { ChatInputInteraction, Command } from 'disploy';
+
+const Ping: Command = {
+	name: 'ping',
+	description: 'pong!',
+
+	run(interaction: ChatInputInteraction) {
+		interaction.reply({
+			content: 'Hello World!',
+		});
+	},
+};
+
+export default Ping;
```

Comment:
```suggestion
export default {
	name: 'ping',
	description: 'pong!',

	run(interaction: ChatInputInteraction) {
		interaction.reply({
			content: 'Hello World!',
		});
	},
} satisfies Command;
```

I see you using `satisfies` a lot in the docs introduction, so here you go.

Reply:
`satisfies` is still an RC feature, it shouldn't be used here in the documentation since it doesn't explicitly say to use the RC version, compared to the framework documentation which expects you to use `create-disploy-app` which already sets this up.